library/template_variables.tmpl

```diff
@@ -0,0 +1,150 @@
+# define template variables
+{{ define `root_dir` -}}
+    {{ .input_root_dir }}
+{{- end }}
+
+{{ define `project_name` -}}
+    {{ .input_project_name }}
+{{- end}}
+
+{{ define `cloud` -}}
```

Comment:
Enum support coming up! We can then remove this. https://github.com/databricks/cli/pull/668

Reply:
MLOPS-194